### ISSUE TITLE
Clear animation prevents new 'duplicate' toast

### ISSUE THREAD

**Issue by kuhnmi** [scttcper/ngx-toastr#342]
If both, fade-out animations and preventDuplicates are enabled, the following sequence of actions won't show the second toast (triggered in step 3):
1. Show toast with message 'test'
2. Clear all toasts
3. Show toast with message 'test' (triggered while the clear animation is still running)

As after step 2 the toasts are 'logically' cleared, I would expect that the second toast is no longer considered a duplicate of the first (cleared) toast. Thus, I would expect the second toast to be shown.

This can be reproduced in the demo (https://scttcper.github.io/ngx-toastr/) if the ease time is chosen long enough such that it is manually possible to trigger the second toast (step 3) while the clear animation is still running.

In my particular use-case, I want to clear a (sticky) error message before submitting a new rest call that might potentially bring up the same error again. I currently work around the problem by making sure the new error toast is only triggered after clear has fully (including clear animation) completed.

If this behaviour is considered a feature, rather than a bug  - would it be possible to make the behaviour configurable?

Note: Similar problems/requirements seem to have popped up with other toast libraries (see e.g. https://github.com/CodeSeven/toastr/issues/118#issuecomment-302412921)

**Comment by masum7** [comment]
This is still an issue.
